src/README.md

```diff
@@ -1,5 +1,90 @@
-<!-- This README file is going to be the one displayed on the Grafana.com website for your plugin -->
+<!-- This README file is going to be the one displayed on the Grafana.com website -->
 
-# Sneller
+# Sneller Grafana Data Source
 
+## Introduction
 
+The official Sneller data source plugin for Grafana can be used to conveniently visualize data from a Sneller database or to create alerts that are triggered when certain conditions are met.
+
+## Installation
+
+Please follow the official documentation on how to install a public data source from the store:
+https://grafana.com/docs/grafana/latest/administration/data-source-management/#add-a-data-source
+
+Search for "Sneller" to find the official Sneller data source.
+
+## Configuration
+
+The Sneller data source provides only the most essential settings for quick and easy setup.
+
+![](./img/readme_config.png)
+
+### Sneller Region
+
+If you are using Sneller Cloud, you can select your home region (e.g. `us-east-1`) from the drop-down menu to automatically configure the endpoint URL.
```

Comment:
Instead of "home region" we should probably use a different term. We use "home region" to refer to where the API tokens live, but really we'd want people to use the region in which the tables they want to query are located.

Reply:
Or "table region"?

Reply:
Or maybe "bucket region"? (Since what we really care about is the location of the sneller "cache bucket")

Reply:
Sounds good :+1 Will change to "bucket region".